flow/Trace.cpp

```diff
@@ -1471,9 +1472,16 @@ void TraceBatch::dump() {
 	buggifyBatch.clear();
 }
 
-TraceBatch::EventInfo::EventInfo(double time, const char* name, uint64_t id, const char* location) {
+TraceBatch::EventInfo::EventInfo(double time,
+                                 double monotonicTime,
+                                 const char* name,
+                                 uint64_t id,
+                                 const char* location) {
 	fields.addField("Severity", format("%d", (int)TRACE_BATCH_IMPLICIT_SEVERITY));
 	fields.addField("Time", format("%.6f", time));
+	// Include monotonic time for computing elapsed time between events on the same machine.
+	// The Time field is based on now(), which doesn't advance between wait()'s.
+	fields.addField("MonotonicTime", format(">%.6f", monotonicTime));
```

Comment:
What does the `>` do here in the format string?

Reply:
Nothing, must be a typo.